strict-version-matcher-plugin/src/main/java/com/google/android/gms/dependencies/DataObjects.kt

```diff
@@ -84,6 +90,16 @@ data class Dependency(val fromArtifactVersion: ArtifactVersion, val toArtifact:
   }
 
   companion object {
+    fun dependencyExclusion(toArtifact: Artifact, toArtifactVersion: String) : Boolean {
+      // The 16 -> 17 major version bump for this library wasn't a breaking change so allow Gradle
+      // to uplift it.
+      if (toArtifact.getGradleRef() == "com.google.firebase:firebase-measurement-connector"  &&
```

Comment:
actually this uplifting is only ok if the requirement did not come from firebase-measurement-connector-impl.  It's a breaking change because it adds a method to the interface which firebase-measurement-connector-impl implements.  Is there a way we can express that?